source/_integrations/voip.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: VoIP
+description: Voice over IP
+ha_category:
+  - Voice
+ha_iot_class: Local Push
+ha_release: '2023.5'
+ha_codeowners:
+  - '@balloob'
+  - '@synesthesiam'
+ha_domain: voip
+ha_integration_type: integration
+ha_quality_scale: internal
+---
+
+The VoIP integration enables users to talk to [Assist](/docs/assist) using an analog phone and a VoIP adapter such as the [Grandstream HT801](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht801).
+
+<p class='img'>
+  <img src="/images/integrations/voip/voip_adapter.png" />
+  Connecting a phone to Home Assistant requires an adapter.
+</p>
+
+## Getting Started
+
+Start by setting up the VoIP integration. In Home Assistant, go to *Settings*, *Devices & Services*, clicking the *Add Integration* button, and entering "voip":
+
```

Comment:
We have my buttons for this, as a matter of fact, there is a standard configuration snippet you can include. This will help standardize our documentation.

```
{% include integrations/config_flow.md %}
```